userspace/libsinsp/filterchecks.cpp

```diff
@@ -90,6 +90,13 @@ int32_t gmt2local(time_t t)
 	return dt;
 }
 
+static inline bool str_match_start(const std::string& val, size_t len, const char* m)
+{
+	return val.compare(0, len, m) == 0;
+}
+
+#define VAL_MATCH(s) str_match_start(val, sizeof (s) -1, s)
```

Comment:
Minor nit that you can ignore--I'd prefer an inline function over a preprocessor define.

Reply:
inline func won't work with sizeof - looks like we will need to do strlen in runtime